backend/__tests__/cronJobs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import ComplianceReport from '../src/db/schemas/compliance';
+import { startCronJobs } from '../src/cronJobs';
+import * as removeExpiredDrafts from '../src/cronJobs/removeExpiredDrafts';
+import { appConfig } from '../src/config';
+import sinon from 'sinon';
+
+describe('Cron Job', () => {
+    let mongod;
+    let clock;
+    let originalCronSchedule = appConfig.cron.removeExpiredDraftsSchedule;
+
+    before(async () => {
+        mongod = await MongoMemoryServer.create();
+        const uri = mongod.getUri();
+
+        if (mongoose.connection.readyState === 1) {
+            await mongoose.disconnect();
+        }
+        await mongoose.connect(uri);
+
+        const currentDate = new Date();
+        const expiredDate = new Date(currentDate);
+        expiredDate.setDate(expiredDate.getDate() - 1);
+        const futureDate = new Date(currentDate);
+        futureDate.setDate(futureDate.getDate() + 1);
+
+        await ComplianceReport.create({
+            softwareName: "openIMIS",
+            logo: "https://openIMIS.com/logo.png",
+            website: "https://openIMIS.com",
+            documentation: ["https://openIMIS.com/doc1", "https://openIMIS.com/doc2"],
+            pointOfContact: "contact@openIMIS.com",
+            status: 0,
+            expirationDate: new Date("2024-11-24T00:00:00.000Z"),
+            compliance: [],
+            link: "12345"
+        });
+        
+        await ComplianceReport.create({
+            softwareName: "openIMIS",
+            logo: "https://openIMIS.com/logo.png",
+            website: "https://openIMIS.com",
+            documentation: ["https://openIMIS.com/doc1", "https://openIMIS.com/doc2"],
+            pointOfContact: "contact@openIMIS.com",
+            status: 0,
+            expirationDate: new Date("2022-11-24T00:00:00.000Z"),
+            compliance: [],
+            link: "123456"
+        });
+
+        clock = sinon.useFakeTimers(currentDate.getTime());
+        appConfig.cron.removeExpiredDraftsSchedule = '* * * * * *';
+        startCronJobs();
+    });
+
+    after(async () => {
```

Comment:
can you add description that its a teardown after tests?

Reply:
I will provide comments:
// Teardown after all tests have been executed.
// This hook is responsible for cleaning up resources
// and restoring configurations to their original state.